components/tool-navbar.vue

```diff
@@ -24,7 +24,7 @@
                         :active="currentPageName === navItem.fullName"
                         :class="getNavItemColor(navItem)"
                         :data-cy="'menu-item-' + navItem.pageName"
-                        :disabled="!isPageAccessible(navItem.pageName)"
+                        :disabled="!isPageAccessible(pageData[getNextPage].pageName)"
```

Comment:
Was this done by accident? This doesn't seem right.

Reply:
Thanks for catching this. There was something funky that happened when I resolved merge conflict for this branch (thus my request for feedback in my second comment on this PR). Fixed.

Reply:
Oh, sorry, didn't see your comments. Looking now!